No compatible version found: uglifyjs@'>=2.3.6 <2.4.0'

`npm install` currently yields an error:

    npm ERR! notarget No compatible version found: uglifyjs@'>=2.3.6 <2.4.0'
    npm ERR! notarget Valid install targets:
    npm ERR! notarget ["2.4.10"]
    npm ERR! notarget
    npm ERR! notarget This is most likely not a problem with npm itself.
    npm ERR! notarget In most cases you or one of your dependencies are requesting
    npm ERR! notarget a package version that doesn't exist.
    npm ERR! notarget
    npm ERR! notarget It was specified as a dependency of 'react-server-rendering-example'
    npm ERR! notarget


Attached `npm-debug.log`

    0 info it worked if it ends with ok
    1 verbose cli [ 'node', '/usr/local/bin/npm', 'start' ]
    2 info using npm@2.7.4
    3 info using node@v0.12.2
    4 verbose run-script [ 'prestart', 'start', 'poststart' ]
    5 info prestart react-server-rendering-example@0.0.1
    6 info start react-server-rendering-example@0.0.1
    7 verbose unsafe-perm in lifecycle true
    8 info react-server-rendering-example@0.0.1 Failed to exec start script
    9 verbose stack Error: react-server-rendering-example@0.0.1 start: `watchify -o browser-bundle.js -d .`
    9 verbose stack Exit status 127
    9 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
    9 verbose stack     at EventEmitter.emit (events.js:110:17)
    9 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:14:12)
    9 verbose stack     at ChildProcess.emit (events.js:110:17)
    9 verbose stack     at maybeClose (child_process.js:1015:16)
    9 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
    10 verbose pkgid react-server-rendering-example@0.0.1
    11 verbose cwd /Users/user1/Desktop/react-server-rendering-example
    12 error Darwin 14.4.0
    13 error argv "node" "/usr/local/bin/npm" "start"
    14 error node v0.12.2
    15 error npm  v2.7.4
    16 error code ELIFECYCLE
    17 error react-server-rendering-example@0.0.1 start: `watchify -o browser-bundle.js -d .`
    17 error Exit status 127
    18 error Failed at the react-server-rendering-example@0.0.1 start script 'watchify -o browser-bundle.js -d .'.
    18 error This is most likely a problem with the react-server-rendering-example package,
    18 error not with npm itself.
    18 error Tell the author that this fails on your system:
    18 error     watchify -o browser-bundle.js -d .
    18 error You can get their info via:
    18 error     npm owner ls react-server-rendering-example
    18 error There is likely additional logging output above.
    19 verbose exit [ 1, true ]

I'm using node v0.12.2.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "browserify": "~2.36.0",
     "envify": "~0.2.0",
     "reactify": "~0.4.0",
-    "uglifyjs": "~2.3.6",
+    "uglifyjs": "2.4.x",
     "watchify": "~0.4.1"
   },
   "scripts": {

